Week2/homework/array-exercises.js

```diff
@@ -0,0 +1,31 @@
+//Write a JavaScript function to get the first element of an array
+
+```js
+const myArray = [0,1,2,3,4,5,6];
+
+const firstElement = arr => arr.shift();
+console.log(firstElement(myArray));
+```
+---
+
+//Passing a parameter 'n' will return the first 'n' elements of the array.
+```js
+const myArray = [0,1,2,3,4,5,6];
+
+const firstnElements = (arr,n) => arr.slice(0,n);
+console.log(firstnElements(myArray,4));
```

Comment:
Try combining the `firstElement` and the `firstnElements` into a single function by using a defualt parameter.
Why is your code in Markdown?

Reply:
I don`t know why I did it.I think i did it  by using a default parameter.Thank you for review.